server/fleet/users.go

```diff
@@ -407,6 +407,9 @@ func saltAndHashPassword(keySize int, plaintext string, cost int) (hashed []byte
 	withSalt := []byte(fmt.Sprintf("%s%s", plaintext, salt))
 	hashed, err = bcrypt.GenerateFromPassword(withSalt, cost)
 	if err != nil {
+		if errors.Is(err, bcrypt.ErrPasswordTooLong) {
+			return nil, "", NewInvalidArgumentError("Could not create user. Password is over the 48 characters limit. If the password is under 48 characters, please check the auth_salt_key_size in your Fleet server config.", "password too long")
```

Comment:
Could we provide the actual limit in the error string calculated from the value of `auth_salt_key_size`?

Reply:
True, but it's noted in the issue that we should hardcode the number 48 for now:
> "48" in the copy is static for now. Based on the default maximum value which comes from 72 bytes - default (24) salt key size.

Reply:
OK, sounds good.